### PR TITLE
fix: update idle onboarding copy

### PR DESCRIPTION
**Pull request by reallybeard**
## Description

- Idle requested we update the copy
- https://idlelabs.notion.site/idlelabs/ShapeShift-copy-UI-ecea511b0f35475b8d874fca3fb8ce83

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #

## Risk

no risk

### Operations

Copy matches the updates per Idle's request.

## Screenshots (if applicable)
Screen Shot 2023-03-09 at 10 51 56 AM
Screen Shot 2023-03-09 at 10 51 59 AM
Screen Shot 2023-03-09 at 10 52 04 AM
Screen Shot 2023-03-09 at 10 52 09 AM
Screen Shot 2023-03-09 at 10 52 15 AM
Screen Shot 2023-03-09 at 10 52 17 AM

### CODE REVIEW

File: src/assets/translations/en/main.json [hunk 1]
```diff
@@ -1768,32 +1785,34 @@
...
       "noLockingPeriod": "No locking period or epochs",
-      "autoReinvest": "Auto-Reinvest accrued tokens"
+      "autoReinvest": "Auto-Reinvest accrued tokens",
+      "fundsCoveredJunior": "Funds covered by Junior Tranches",
+      "automation": "Automation and optimization of the yield generation across Aave, Compound, and multiple Senior Tranches"
     },
     "bestYield": {
       "title": "What is Best Yield?",
       "body": "Automatically maximize yield across multiple battle-tested DeFi protocols. Best-Yield strategy optimizes yield generation by socializing gas costs, automating yield generation by algorithmically shifting capital as new yield opportunities arise."
     },
```
Comment (gomesalexandre) [inline src/assets/translations/en/main.json:1808]: Should this go away in favor of Junior/Senior Best Yield as well?

**Review by gomesalexandre — APPROVED** [review]
LGTM, added one question comment

File: src/assets/translations/en/main.json (hunk 1 above)
Comment (reallybeard) [reply src/assets/translations/en/main.json:1808]: We are still using these in another place that isn't related to updating the copy here. So pull them out as we change stuff I think.

Comment (gomesalexandre) [reply src/assets/translations/en/main.json:1808]: Sounds good to me, let's get this in then

**Pull request merged**